fendermint/vm/interpreter/Cargo.toml

```diff
@@ -28,11 +28,19 @@ fvm_ipld_blockstore = { workspace = true }
 fvm_ipld_encoding = { workspace = true }
 fvm_ipld_car = { workspace = true }
 
-[dev-dependencies]
-quickcheck = { workspace = true }
+futures-core = "0.3.28"
+futures-util = "0.3.28"
+libipld = { version = "0.14", default-features = false, features = ["dag-cbor"] }
 tokio = { workspace = true }
+tokio-stream = "0.1.14"
+tokio-util = {version = "0.7.8", features = ["compat"]}
+pin-project = "1.1.2"
 
+[dev-dependencies]
+quickcheck = { workspace = true }
+fvm = { version = "~3.3", default-features = false, features=["arb", "testing"] }
```

Comment:
There definitely is an `fvm` already in the workspace. Far as I know every different feature combination results in a different compilation, so it's better to use the same. I understand this requires the `testing` feature that isn't needed in the normal context, but look, you can take the version from the workspace and add a feature here:

https://github.com/consensus-shipyard/fendermint/blob/master/fendermint/eth/api/Cargo.toml#L38

More generally, I think we should maintain all versions in the root and avoid mixing. I have a fear like it will end up being a mess in the future.

Reply:
updated